inject.js

```diff
@@ -613,6 +623,21 @@
     parentElement.addEventListener('mousemove', startDragging);
   }
 
+  function handleReposition(video, controller) { // 481_NEW: handling reposition
+    const shadowController = controller.shadowRoot.querySelector('#controller');
+    // Find nearest parent of same size as video parent.
+    var parentElement = controller.parentElement;
```

Comment:
Silly question, why is this block necessary? We don't use parentelElement below?

Reply:
Sorry the the super late response, I forgot about this contribution. This does seem redundant, we could probably just use the parentElement below as you've said

Reply:
Is there a particular way that you'd like me to refactor this code?